packages/expo-router/src/static/registerRootComponent.ts

```diff
@@ -0,0 +1,22 @@
+import type { ComponentType, JSX, PropsWithChildren } from 'react';
+// @ts-expect-error: TODO(@kitten): Define this type (seems to differ from react-native)
+import { AppRegistry } from 'react-native-web';
+
+import type { ExpoRootProps } from '../ExpoRoot';
+import type { RequireContext } from '../types';
+
+type InitialProps = {
+  location: URL;
+  context: RequireContext;
+  wrapper: ComponentType<PropsWithChildren>;
+};
+
+const APP_KEY = 'App';
+
+export function registerStaticRootComponent<P extends InitialProps>(
```

Comment:
The idea here was to be as similar to `registerRootComponent()` from `packages/expo/src/launch/registerRootComponent.tsx` as possible.

Reply:
That's a nice find! Makes a lot of sense to match that